.eslintrc.cjs

```diff
@@ -9,7 +9,7 @@ module.exports = {
     'plugin:react-hooks/recommended',
     'prettier',
   ],
-  ignorePatterns: ['*.css'],
+  ignorePatterns: ['**/*.css', '**/*.json', '**/*.d.ts', '**/vite.config.ts'],
```

Comment:
Combine all the ignorePatterns here from lower configurations

Reply:
Should have all now